### DISCUSSION TITLE
Istioctl operations without IstioOperator CR [IstioOperator removal in v1.21]

### DISCUSSION THREAD

**Discussion by ivan-villani** [istio/istio#51629] [General]
Hi all,

I have wrote an issue, regarding istioctl's binary usage and documentation: https://github.com/istio/istio/issues/51628
Now I am writing with more questions on future plans for the istioctl operations, after the removal of IstioOperator CR and CRD in v1.21:

1. Will there be any changes to the configuration approach when installing Istio with `istioctl install -f`, since it is mentioned ([here](https://github.com/istio/istio/issues/44814)) the planned removal of `kind: IstioOperator`? Will the currently supported `IstioOperator.yaml` file be replaced with something like `values.yaml` file or something different?
2. Will `verify-install` command and/or `--verify` flag be supported in the future?
    - Will the other configuration approach from `Question 1`, if such, will be supported also for the `verify-install` command?
3. Do you plan any changes in istioctl’s `manifest`, `proxy-config` and `proxy-status` commands, related to the absence of IstioOperator CR and CRD?

**Comment by howardjohn** [comment]
Thanks for the verify-install mentioned; that looks broken. I opened https://github.com/istio/istio/issues/51666.

1. `istioctl install` will not change. Just `IstioOperator` is no longer copied into the cluster. Before, it was just written there but did nothing at all. It is still the input to `istioctl install`
2. TBD, follow the issue above for updates :slightly_smiling_face: 
3. Only `manifest` uses the IstioOperator in anyway. Same as (1) applies there.
